packages/spector/src/actions/server-test.ts

```diff
@@ -194,42 +142,61 @@ export async function serverTest(scenariosPath: string, options: ServerTestOptio
       const obj: ServerTestsGenerator = new ServerTestsGenerator(name, api, baseUrl);
       try {
         await obj.executeScenario();
-        success_diagnostics.push({
-          scenario_name: name,
-          status: "success",
-          message: "executed successfully",
+        successfullScenarios.push({
+          name,
+          pathlikeName,
         });
       } catch (e: any) {
-        failure_diagnostics.push({
-          scenario_name: name,
-          status: "failure",
-          message: `code = ${e.code} \n message = ${e.message} \n name = ${e.name} \n stack = ${e.stack} \n status = ${e.status}`,
-        });
+        if (e instanceof ValidationError) {
+          failureDiagnostics.push({
+            scenarioName: name,
+            message: [
+              `Validation failed: ${e.message}:`,
+              ` Expected:\n    ${inspect(e.expected)}`,
+              ` Actual:\n    ${inspect(e.actual)}`,
+            ].join("\n"),
```

Comment:
Nice!

Reply:
yeah, was a bit of a pain before, this is the same message displayed on the server side as well when it reject a request